### PR TITLE
fixed bug; added tests

### PR DESCRIPTION
**Pull request by kcibul**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java [hunk 1]
```diff
@@ -128,14 +128,14 @@ public String getColumnValue(final VariantContext variant) {
                 // Try to use non-AS version
                 // TODO: it looks like the AS_RAW version also trims to a single decimal point??
                 // e.g. MQRankSum=1.465 and turn it into |1.465,1|
-                String outNotAlleleSpecific = getAttribute(variant, GATKVCFConstants.READ_POS_RANK_SUM_KEY, null);
+                String outNotAlleleSpecific = getAttribute(variant, GATKVCFConstants.MAP_QUAL_RANK_SUM_KEY, null);
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java:131]: this is legendary now

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java [hunk 2]
```diff
@@ -257,8 +257,8 @@ public String getColumnValue(final VariantContext variant) {
     AS_SB_TABLE { // Required
         public String getColumnValue(final VariantContext variant) {
             String out = getAttribute(variant, GATKVCFConstants.AS_SB_TABLE_KEY, null);
-            String outNotAlleleSpecific = variant.getGenotype(0).getExtendedAttribute(GATKVCFConstants.STRAND_BIAS_BY_SAMPLE_KEY, null).toString();
             if (out == null) {
```
Comment (RoriCremer) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java:260]: what does this switch do?

Comment (RoriCremer) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java:260]: oh---was this a different bug?

**Review by RoriCremer — APPROVED** [review]

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java (hunk 2 above)
Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnum.java:260]: this was more that we were trying to parse out the "fallback" (non-AS) annotation even if we didn't need it.  Now it just parses it if we can't find the first one.  The tests caught it

**Pull request merged**